src/routes/Board/Board.tsx

```diff
@@ -40,8 +42,12 @@ function Board() {
             <>    
                 {waitingUser}
                 <BoardComponent name={state.board.data!.name} boardstatus={boardstatus}>
-                        {state.board.data!.columns.map((column:any) => (
-                            <Column color="pink" key={column.id} columnId={column.id}/>
+                        {state.board.data!.columns.map((column) => (
```

Comment:
Maybe we should create a Column type

Reply:
Column type is used here since `board.data.columns` is already typed